examples/pytorch/text-classification/Fine Tune BERT LLM.ipynb

```diff
@@ -0,0 +1,683 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    good_review = "This is one of the best restaurants. I've ever been to."
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kubeflow/training-operator/pull/2021/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
It makes sense. 
Also, Grammarly says this is fine as well :)